src/content.ts

```diff
@@ -0,0 +1,56 @@
+import { Bee } from '@ethersphere/bee-js'
+import client from 'prom-client'
+import type { ContentConfig } from './config'
+import { logger } from './logger'
+import { register } from './metrics'
+
+const contentReuploadCounter = new client.Counter({
+  name: 'content_reupload_counter',
+  help: 'How many pinned content items were uploaded',
+})
+register.registerMetric(contentReuploadCounter)
+
+export class ContentManager {
+  private interval?: ReturnType<typeof setInterval>
+  private isReuploading?: boolean = false
+
+  public async refreshContentReupload(beeApi: Bee): Promise<void> {
+    const pins = await beeApi.getAllPins()
+
+    if (!pins.length) {
+      logger.info(`no pins found`)
+    } else {
+      logger.info(`checking pinned content (${pins.length} pins)`)
+      for (let i = 0; i < pins.length; i++) {
+        const pin = pins[i]
```

Comment:
Ehm, this is the pre-historic way how to iterate an array in JS ;-) Thanks to new ECMAScripts standards we have the [`for ... of`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) (which I have originally referenced in the comment) for looping over iterable (which array is). So please refactor it to that.

Reply:
Sorry for this, too much JS in my veins

Reply:
No prob 🙃